src/Parser/Priest/Discipline/Normalizers/AtonementSuccessiveDamage.js

```diff
@@ -0,0 +1,45 @@
+import EventsNormalizer from 'Parser/Core/EventsNormalizer';
+
+import isAtonement from '../Modules/Core/isAtonement';
+
+class AtonementSuccessiveDamage extends EventsNormalizer {
+
+  normalize(events) {
+    const fixedEvents = [];
+    const _damageEventIndexes = [];
+
+    let _encounteredTargetIDs = [];
+
+    events.forEach((event, eventIndex) => {
+
+      fixedEvents.push(event);
+
+      if(event.type === "damage" && event.sourceIsFriendly && !event.targetIsFriendly){
+        _damageEventIndexes.push(eventIndex);
+        _encounteredTargetIDs = [];
+        return;
+      }
+
+      if(event.type === "heal" && isAtonement(event)) {
+
+        // We encountered a targetID we already encountered since the last damage
+        // event. We push down the last damage event here
+        if(_encounteredTargetIDs.indexOf(event.targetID) >= 0) {
```

Comment:
Since this doesn't check that the sourceId matches, I think this will break if you have 2 disc priests in the group.

Reply:
the atonement from another priest shows here ? When I look at the initial events I only have events that are from the priest. What am I not understanding ?

Reply:
Being healed by another Disc shows up.